crowdfunding_compassion/forms/project_creation_form.py

```diff
@@ -213,7 +213,7 @@ def _form_fieldsets(self):
             ],
         }]
         storage = self.wiz_storage_get().get('steps')
-        if self.main_object or storage.get(2, {}).get("type") == "collective":
+        if storage.get(1, {}).get("type") == "collective" or self.main_object:
```

Comment:
I don't think this is a good change, we need values of the step1 form and not step2. Why taking the storage in priority to the main_object ? I think you should revert this change.